### DISCUSSION TITLE
Welcome to htmldoc Discussions!

### DISCUSSION THREAD

**Discussion by michaelrsweet** [michaelrsweet/htmldoc#438] [Announcements]
👋 Welcome!
==========

Discussions is a place to connect with other members of the HTMLDOC community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by senthilk99** [comment]
HI, I am Senthilkumar S, PM.

**Comment by senthilk99** [comment]
Would like to know the  source files for installing on RHEL 8. 0and we unfortunately can not install snapstore on RHEL 8.0.

**Comment by smartphoneworld** [comment]
Hi Michael.. this is Mahammad Mostafa, a software developer with more than 10 years experience. I contributed to several open sources; I wounder if you have any open contribution tasks.
I am experienced with htmlDoc. Last person I worked with on htmlDoc with was Andrew Schmadeke (Focus School Software).

**Reply by michaelrsweet** [reply]
Hi Mahammad, I don't have any specific tasks for the HTMLDOC project you could work on. Two related projects - [htmlcss](https://github.com/michaelrsweet/htmlcss) and [PDFio](https://github.com/michaelrsweet/pdfio) - might form the basis of a new version of HTMLDOC supporting CSS, and you might find things you could contribute there?

**Comment by smartphoneworld** [comment]
Thank you for the information, Michael. I can see there are many open issues and feature requests for PDFio. Is there a prioritized list available?

**Reply by michaelrsweet** [reply]
The priority tags are the only mechanism we have available with Github issues.

**Reply by smartphoneworld** [reply]
I can see one enhancement with the tag "priority-high" which is "Add multiple page content stream". I suppose this is the way to proceed. How should I start? Who will I be reporting to? Should I start with the main stream or by some branch? Has anyone else worked it out?

**Reply by michaelrsweet** [reply]
The easiest way is to fork PDFio and work from your own repository. Once you have something for me to look at, create a pull request and describe the changes.

Nobody has been working on this yet, but if you come up with a planned API you can run it past me in the corresponding issue and then I can provide feedback before you do a lot of work.
